srcpkgs/recoll/template

```diff
@@ -1,18 +1,32 @@
 # Template file for 'recoll'
 pkgname=recoll
-version=1.30.0
+version=1.32.7
 revision=1
 build_style=gnu-configure
 build_helper=qmake
```

Comment:
```suggestion
 build_helper="qmake python3"
```

ok i just merged a build helper that does the big crossbuild block i suggested earlier. rebase and add the build helper and removing the big block should be possible

Reply:
done